charters/IngestSupport/charter.md

```diff
@@ -0,0 +1,100 @@
+# Ingest Support
+
+## Description
+
+The Ingest Support team acts as a liaison between labs who want to contribute data to the Human Cell Atlas (HCA) and the Data Coordination Platform (DCP). This team works with data contributing labs, to understand each lab's data and ensure the experiments are well-represented in the platform and to support the lab writing data to the platform, working with them to resolve any issues in the process.
+
+## Definitions
+
+*Data contributor* - any user who uploads data and metadata into the DCP
+
+*Data wrangler* - a member of the ingest support team who supports to data contributors wanting to upload data into the DCP
+
+*Submission* - the grouping of data and metadata that makes sense to a data contributor. For example, a data contributor might typically upload all the data at once for a publication they have in preparation. This grouping does not necessarily correspond to how data is stored or displayed in the DCP and how it should be displayed to a data consumer
```

Comment:
I'd move *Submission* after *Data* and *Metadata* since it relies on those definitions.